GearRent/ViewControllers/CreateListingViewController.m

```diff
@@ -11,13 +11,15 @@
 #import "ProfileImagePickerViewController.h"
 #import "UIImageView+AFNetworking.h"
 #import "TimeInterval.h"
-#import "Item.h"
+#import "Listing.h"
 #import "Reservation.h"
 #import "MapKit/MapKit.h"
 #import "CoreLocation/CoreLocation.h"
 #import "GNGeoHash.h"
+#import "APIManager.h"
+#import "Category.h"
 
-@interface CreateListingViewController ()<UICollectionViewDataSource, UICollectionViewDelegate, JTCalendarDelegate,ProfileImagePickerViewControllerDelegate, MKMapViewDelegate, CLLocationManagerDelegate>
+@interface CreateListingViewController ()<UICollectionViewDataSource, UICollectionViewDelegate, JTCalendarDelegate,ProfileImagePickerViewControllerDelegate, MKMapViewDelegate, CLLocationManagerDelegate, UIPickerViewDelegate, UIPickerViewDataSource>
```

Comment:
This View Controller is starting to have a lot of responsibilities. It might be worth refactoring some of that behavior out into new classes. For example, you can have a `CreateListingPickerViewManager` which conforms to the `UIPickerViewDelegate` and `UIPickerViewDataSource` and you just hold an instance of that manager as a private variable in `CreateListingViewController`.

Reply:
Yeah, this is definitely something I willl due after MVP